app/views/includes/scripts.html

```diff
@@ -1,9 +1,7 @@
 <script src="{{ globals.basePath }}/public/javascripts/govuk-frontend/govuk/all.js"></script>
 <script nonce="{{ globals.nonce }}">window.GOVUKFrontend.initAll()</script>
 {% if (settings.env == 'prod') or (settings.env == 'production') %}
-    <script src="https://js-cdn.dynatrace.com/jstag/17177a07246/bf00910jpo/6d0dc4b49e2f265a_complete.js" integrity="sha384-rsf2+Rs7JDFenI0EJQ9zwfXObRU9MbEKFgsma7tGrIUjhNwoFOfjAd6Vtsnox0YH" crossorigin="anonymous"></script>
-{% else %}
-    <input type="hidden" id="environment" name="environment" value={{ settings.env }}>
+    <script src="https://js-cdn.dynatrace.com/jstag/17177a07246/bf00910jpo/6d0dc4b49e2f265a_complete.js"></script>
```

Comment:
you need to make sure the integrity="sha384-rsf2+Rs7JDFenI0EJQ9zwfXObRU9MbEKFgsma7tGrIUjhNwoFOfjAd6Vtsnox0YH" is set.